security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_generate_conf.php

```diff
@@ -250,8 +250,8 @@
 
 /* def perform_stat */
 
-if (!empty($config['installedpackages']['snortglobal']['stats_log_limit_size']) && $config['installedpackages']['snortglobal']['stats_log_limit_size'] != "0")
-	$stats_log_limit = "max_file_size " . $config['installedpackages']['snortglobal']['stats_log_limit_size'] * 1000;
+if (!empty(config_get_path('installedpackages/snortglobal/stats_log_limit_size')) && config_get_path('installedpackages/snortglobal/stats_log_limit_size') != "0")
```

Comment:
```suggestion
if (config_get_path('installedpackages/snortglobal/stats_log_limit_size', '0') != '0')
```

Reply:
Modified code as suggested to use a default return value.